setup.py

```diff
@@ -57,6 +57,10 @@
             "pytest>=6.1.1,<8",
             "pandas",
         ],
+        "pgvector": [
```

Comment:
Optional: add doc in https://microsoft.github.io/autogen/docs/installation/Optional-Dependencies#retrievechat
cc @thinkall for considering bigger documentation change in https://microsoft.github.io/autogen/docs/topics/retrieval_augmentation

Reply:
I have added a few updates to the docs.